registrar/apps/api/v1/views.py

```diff
@@ -285,7 +285,9 @@ def get(self, request, *args, **kwargs):
         """
         Submit a user task that retrieves program enrollment data.
         """
-        return self.invoke_download_job(list_program_enrollments, self.program.key)
+        include_username_email = waffle.flag_is_active(request, 'include_name_email_in_get_program_enrollment')
+        return self.invoke_download_job(
+            list_program_enrollments, self.program.key, include_username_email)
```

Comment:
```suggestion
        return self.invoke_download_job(
            list_program_enrollments, self.program.key, include_username_email
        )
```
Style nit based on [our Python style guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines).